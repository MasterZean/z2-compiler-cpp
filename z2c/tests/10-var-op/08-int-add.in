{ val a = Int{1} + Bool{2} }
{ val a = Int{1} + Small{2} }
{ val a = Int{1} + Byte{2} }
{ val a = Int{1} + Short{2} }
{ val a = Int{1} + Word{2} }
{ val a = Int{1} + Int{2} }
{ val a = Int{1} + DWord{2} }
{ val a = Int{1} + Long{2} }
{ val a = Int{1} + QWord{2} }
{ val a = Int{1} + Float{2} }
{ val a = Int{1} + Double{2} }
{ val a = Int{1} + Char{2} }
{ val a = Int{1} + PtrSize{2} }
