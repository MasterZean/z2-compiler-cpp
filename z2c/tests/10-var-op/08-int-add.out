{
	int32 a = (int32)(1) + (bool)(2);
}
{
	int32 a = (int32)(1) + (int8)(2);
}
{
	int32 a = (int32)(1) + (uint8)(2);
}
{
	int32 a = (int32)(1) + (int16)(2);
}
{
	int32 a = (int32)(1) + (uint16)(2);
}
{
	int32 a = (int32)(1) + (int32)(2);
}
{
	uint32 a = (int32)(1) + (uint32)(2);
}
{
	int64 a = (int32)(1) + (int64)(2);
}
{
	uint64 a = (int32)(1) + (uint64)(2);
}
{
	float a = (int32)(1) + (float)(2);
}
{
	double a = (int32)(1) + (double)(2);
}
{
	uint32 a = (int32)(1) + (uint32)(2);
}
{
	size_t a = (int32)(1) + (size_t)(2);
}
